RedHatInsights/insights-operator-controller
# Add Makefile

# Description

Fixes # (issue)
CCXDEV-366

## Type of change
Add Makefile and update travis.yml file accordingly

Please delete options that are not relevant.
Not sure here... :) probably a new feature. 

## Testing steps
This should be tested automatically by Travis CI. Otherwise you can try to execute the make commands locally.
You can see at https://github.com/tremes/insights-operator-controller/tree/my-master that the CI passed.
